brave/brave-core
# Storybook: Created Wallet Onboarding Views

Resolves <https://github.com/brave/brave-browser/issues/15597>

Created the Wallet Onboarding Views and added it to storybook under Concepts/Components.

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. `npm run storybook`
2. Confirm that storybook shows the new component.
3. Check functionality of the new component.


https://user-images.githubusercontent.com/40611140/116940657-7fa4a580-ac2b-11eb-8db6-0975eb60ab5c.mov
